Quadriga/src/main/webapp/WEB-INF/spring/spring-security.xml

```diff
@@ -1,81 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<beans:beans xmlns="http://www.springframework.org/schema/security"
-	xmlns:beans="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xmlns:context="http://www.springframework.org/schema/context"
-	xsi:schemaLocation="http://www.springframework.org/schema/beans
-	http://www.springframework.org/schema/beans/spring-beans-3.2.xsd
-	http://www.springframework.org/schema/security
-	http://www.springframework.org/schema/security/spring-security-4.0.xsd">
-	<http pattern="/rest/**" auto-config="true" use-expressions="true" create-session="stateless">
-		<intercept-url pattern="/rest/**" access="hasAnyRole('ROLE_QUADRIGA_USER_ADMIN','ROLE_QUADRIGA_USER_STANDARD','ROLE_QUADRIGA_USER_COLLABORATOR')"/>
-		<http-basic/>		
-		<!-- <custom-filter ref="digestFilter" after="BASIC_AUTH_FILTER" /> -->
-		<csrf disabled="true"/>
-	</http>
-	<http pattern="/public/**" auto-config="true" use-expressions="true" create-session="stateless">
-	   <intercept-url pattern="/public/**" access="permitAll"/>
-	   <http-basic/>       
-       <csrf disabled="true"/>
-	</http>
```

Comment:
what about this http definition? It doesn't seem to have been added to the java config?